### DISCUSSION TITLE
can't understand how to get NFT

### DISCUSSION THREAD

**Discussion by gulshanpr** [Cyfrin/foundry-full-course-cu#64] [Q&A] [answered]
when i visit sepolia to get NFT, i don't understand how i can solve the challenge. I looked at the contract section but couldn't understand.

**Comment by heysourin** [comment, marked as answer]
Hey @gulshanpr 

1. Go to `Write Contract`
![Screenshot 2023-06-13 200319](https://github.com/Cyfrin/foundry-full-course-f23/assets/98945276/67d4b12b-36a2-4466-8475-8be36bbfc82a)

2. Then interact with the contract by connecting your wallet and adding the correct data
![2](https://github.com/Cyfrin/foundry-full-course-f23/assets/98945276/f4c1da52-5f36-4629-8e42-bc2666388d01)

**Reply by alymurtazamemon** [reply]
@gulshanpr Also above the `solveChallenge` function in the code section you will find the instructions for each challenge.

**Reply by cromewar** [reply]
Thanks @heysourin, please @gulshanpr let us know if this was helpful.

**Reply by gulshanpr** [reply]
Thanks @heysourin and @alymurtazamemon for helping me out. @cromewar yes I got my answer and I minted a NFT on sepolia too.

**Reply by PatrickAlphaC** [reply]
they are not all this easy 😈

**Comment by nik132-eng** [comment]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/77881638/3064f605-a3b4-43a9-a309-27f3958ef88f)


It shows this, I completed lesson 1

**Reply by gulshanpr** [reply]
you might have minted NFT for this lesson already, that's why it is showing this.

**Reply by nik132-eng** [reply]
> you might have minted NFT for this lesson already, that's why it is showing this.
> 

No I checked that

In the open sea test net on my profile this nft not showing

**Reply by PatrickAlphaC** [reply]
Or you didn't solve it 😈

That's ok! You can always come back as your skills increase! You will defeat this challenge in the future ;)
